### ISSUE TITLE
Changing color doesn't seem to trigger redraw/update

### ISSUE THREAD

**Issue by wolfv** [bloomberg/bqplot#788]
When changing the color trait, it doesn't seem to immediately trigger a redraw.

If changing e.g. x/y coordinates after color, it redraws right away with the correct color. 

Otherwise, need to hover or wait.

**Comment by wolfv** [comment]
When running the binder / scatter example, adding the following after the basic example:

```
scatt.color = [5] * 30
```

shows the bug.

**Comment by maartenbreddels** [comment]
I guess you mean:
```
scatt.colors = ['red'] * len(scatt.x)
```
note that color and colors exist ;)

**Comment by martinRenou** [comment]
This seems to be fixed by #798. Would you mind closing it @wolfv ?
